source/components/progress-bar/progress-bar.styl

```diff
@@ -0,0 +1,61 @@
+/* ==========================================================================
+   Variables
+========================================================================== */
+
+$c-background-bar = lightgrey;
+
+/* ==========================================================================
+   ProgressBar Elements
+========================================================================== */
+
+.progressWrap {
+   margin: 0;
+   padding: 0;
+   border-radius: 15px;
+   display: flex;
+	width: 100%;
+	height: 4px;
+   background-color: $c-background-bar;
+	position: relative;
+}
+
+
+.progressBar {
+   margin: 0;
+   padding: 0;
+   display: flex;
+   position: relative;
+   box-shadow: inset 0 2px 9px rgba(255, 255, 255, 0.3), inset 0 -2px 6px rgba(0, 0, 0, 0.4);
+   overflow: hidden;
+	height: 4px;
+   animation: progressBar linear 3s both;
```

Comment:
Essa prop animation funiciona bem! Mas acaba que, como ela tem inicio e fim, não anima quando muda a porcentagem da barra depois da primeira vez! Pra fazer isso, é melhor usar a `transition`! Por exemplo:
```css
transition: width .8s ease-in-out;
```
Ou seja, fazer uma transição apenas para o atributo "width", que vai demorar 0.8s e vai ser da forma ease-in-out.